common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -165,20 +173,24 @@ object UserPreferencesQueries {
           )
         r   <-
           OptionT
-            .liftF[F, SortedMap[react.gridlayout.BreakpointName, (Int, Int, Layout)]] {
-              query[F](c).map { r =>
+            .liftF[F, (Int, SortedMap[react.gridlayout.BreakpointName, (Int, Int, Layout)])] {
+              query[F](uid.show, c, resizableArea.value).map { r =>
+                val w = r.explore_resizable_width_by_pk.map(_.width).getOrElse(defaultValue._1)
                 if (r.grid_layout_positions.isEmpty) defaultValue
```

Comment:
When there's a defined `width` but not a `layout`, this seems to override the defined `witdh` with the default. Is this the intended behavior?

Reply:
Thank you for pointing this out, I added a fix for this